pallene/gc.lua

```diff
@@ -49,7 +49,7 @@ function gc.compute_stack_slots(func)
 
     for i, cmd in ipairs(flat_cmds) do
         for _, val in ipairs(ir.get_srcs(cmd)) do
-            if val._tag == "ir.Value.LocalVar" then
+            if val._tag == "ir.Value.LocalVar" or val._tag == "ir.Value.Upvalue" then
```

Comment:
That said, I suppose it wouldn't hurt to mention this in that comment at top of the `gc.lua`.

Reply:
>  Because the upvalues are stored in the function closure then if the function closure is reachable by the garbage collector 
Ah yeah, I somehow missed that 👍